### DISCUSSION TITLE
Global or Shared Favourites

### DISCUSSION THREAD

**Discussion by paulmortlock** [sam-ludlow/mame-ao#37] [Ideas]
I can assign a username and choose to share my Favourites with other using mame-ao

I would like that feature and would like to contribute as I am .net developer.

**Comment by sam-ludlow** [comment]
Hello @paulmortlock 

Hmmm this would require a server with a user DB and stuff.

Sounds like fun to code just not sure how many people would bother to login?

But yes I sort of like the idea of sharing favourites, if anyone could be bothered.

I was thinking of adding some sort of "phone home" with stats of what people are actually playing and for how long. Then display all this usage stuff on a web.

Hi scores would also be nice, not sure where they live in MAME? I'll look into that.

Cheers
Sam

**Reply by paulmortlock** [reply]
Yes that would be so good. Do you know how many users actually use this. I posted a link to the Github on Reddit but I got banned from r/mame for violating the rules! Hi scores would be awesome,

**Comment by paulmortlock** [comment]
https://docs.mamedev.org/plugins/hiscore.html

**Comment by sam-ludlow** [comment]
Lolz I got banned from r/mame for the same thing a while back, thought the MAME community might appreciate it but it is forbidden apparently. I get the piracy thing but MAME-AO is just software not a ROM repository.

Anyway even though the reddit posts get removed after a few hours they still create a GitHub traffic spike. I guessing this was you?
![image](https://github.com/sam-ludlow/mame-ao/assets/47115020/3b719cec-9723-46fa-9bde-9c5a494a8d97)

I would like to promote MAME-AO more but it seems a tricky business. I tried messaging a few YouTube channels but no joy. One YouTube appeared naturally https://www.youtube.com/watch?v=5ny-InKFQXQ that seems to be doing alright.

Anyone feel free to promote MAME-AO, I did it to make playing MAME easy for everyone but getting the message out is another matter.

Might be worth a try on r/Roms. Probably get shot down by reddit users but I can't see how it would be removed. Thing is with users on ROM sites/forums is they are stuck in their ways they like messing about with clrmamepro and don't want to change.
